File: src/main/java/com/trade/securities/domain/EtfTradingInfo.java
package com.trade.securities.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class EtfTradingInfo {
    private String basDd; // 기준일자
    private String isuCd; // 종목코드
    private String isuNm; // 종목명
    private String tddClsprc; // 종가
    private String cmpprevddPrc; // 대비
    private String flucRt; // 등락률
    private String nav; // 순자산가치(NAV)
    private String tddOpnprc; // 시가
    private String tddHgprc; // 고가
    private String tddLwprc; // 저가
    private String accTrdvol; // 거래량
    private String accTrdval; // 거래대금
    private String mktcap; // 시가총액
    private String invstasstNetasstTotamt; // 순자산총액
    private String listShrs; // 상장주식수
    private String idxIndNm; // 기초지수 지수명
    private String objStkprcIdx; // 기초지수 종가
    private String cmpprevddIdx; // 기초지수 대비
    private String flucRtIdx; // 기초지수 등락률
}

File: src/main/kotlin/com/trade/securities/domain/EtfTradingInfo.kt
package com.trade.securities.domain

data class EtfTradingInfo(
    val basDd: String? = null, // 기준일자
    val isuCd: String? = null, // 종목코드
    val isuNm: String? = null, // 종목명
    val tddClsprc: String? = null, // 종가
    val cmpprevddPrc: String? = null, // 대비
    val flucRt: String? = null, // 등락률
    val nav: String? = null, // 순자산가치(NAV)
    val tddOpnprc: String? = null, // 시가
    val tddHgprc: String? = null, // 고가
    val tddLwprc: String? = null, // 저가
    val accTrdvol: String? = null, // 거래량
    val accTrdval: String? = null, // 거래대금
    val mktcap: String? = null, // 시가총액
    val invstasstNetasstTotamt: String? = null, // 순자산총액
    val listShrs: String? = null, // 상장주식수
    val idxIndNm: String? = null, // 기초지수 지수명
    val objStkprcIdx: String? = null, // 기초지수 종가
    val cmpprevddIdx: String? = null, // 기초지수 대비
    val flucRtIdx: String? = null // 기초지수 등락률
)
